aiohttp_xmlrpc/client.py

```diff
@@ -62,7 +63,8 @@ def _parse_response(body, method_name):
             if log.getEffectiveLevel() <= logging.DEBUG:
                 log.debug("Server response: \n%s", body.decode())
 
-            response = etree.fromstring(body)
+            parser = etree.XMLParser(huge_tree=ServerProxy.huge_tree)
```

Comment:
It's instance specific property. You have to remove staticmethod and use `self` here